src/System.Private.CoreLib/shared/System/String.cs

```diff
@@ -444,6 +444,19 @@ public static bool IsNullOrEmpty(string value)
             return (value == null || 0u >= (uint)value.Length) ? true : false;
         }
 
+        [System.Runtime.CompilerServices.IndexerName("Chars")]
+        public char this[Index index]
+        {
+            get
+            {
+                int actualIndex = index.GetOffset(_stringLength);
```

Comment:
Pretty much everywhere else we use `Length` rather than `_stringLength`.  Can we do the same here?

Reply:
I fixed all places as @jkotas  pointed before but it looks I missed this one. Thanks for the catch.